ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/traffic-policies.yml

```diff
@@ -27,6 +27,9 @@ traffic_policies:
       default_actions:
         ipv4:
           drop: true
+      counters:
```

Comment:
blocking PR to get a review from @colinmacgiolla

Reply:
lgtm